src/web/layouts/StandardLayout.tsx

```diff
@@ -540,14 +542,26 @@ export const StandardLayout = ({ CAPI, NAV, format, palette }: Props) => {
 					<GridItem area="lines">
 						<div css={maxWidth}>
 							<div css={stretchLines}>
-								<GuardianLines
-									count={decideLineCount(format.design)}
-									palette={palette}
-									effect={decideLineEffect(
-										format.design,
-										format.theme,
-									)}
-								/>
+								{format.theme === Special.Labs ? (
+									<>
+										<Lines count={1} effect="straight" />
+										<div
+											css={css`
+												height: ${space[2]}px;
+											`}
+										/>
+										<Lines count={1} effect="dotted" />
+									</>
+								) : (
+									<GuardianLines
```

Comment:
Does GuardianLines not support Labs now? Also cc: @jlieb10 who I think may have picked up the card related to this.

Reply:
I think we introduced a [`color` prop](https://github.com/guardian/source/blob/main/src/editorial/web/components/lines/README.md#color) with Labs in mind. It would be good to know if it's valuable.

Reply:
@jfsoul Please see updated PR here https://github.com/guardian/dotcom-rendering/pull/3056

GuardianLines would support labs with some tinkering, but we discovered that we don't need GuardianLines in the longer term because, as @SiAdcock is pointing out, 'Lines' does everything that 'GuardianLines' does - so rather than re-write GuardianLines, it would be better to use this as a starting point to start going back to just using 'Lines'

As for the question about actually using the color prop, I am unsure what designs require it.

Reply:
Sorry I actually got those the wrong way around, I thought `GuardianLines` was from source.